packages/components/src/components/Avatar/index.tsx

```diff
@@ -21,6 +21,7 @@ export const AvatarContainer = styled(Element).attrs({ as: 'span' })(
     height: 8,
     width: 8,
     position: 'relative',
+    flexShrink: 0, // avatars should never shrink inside flex
```

Comment:
flex makes its children shrink when there's not enough space, an avatar should never shrink though

i'm slightly conflicted about the location of this rule. Should it be in a Stack? The stack has rules for elements it will shrink and not; Or should it be in the Avatar which controls whether it will shrink no matter where it is put 🤷‍♂ 

@SaraVieira thoughts?

Reply:
I think it makes sense to be in here since this a very specific case, I am good with having it in here

Reply:
here - Stack or Avatar?

Reply:
I think it should be in `Avatar`.
You also can't fix this inside the parent I'm afraid, it's just how `flex` works

Reply:
I meant nested inside the Stack

pseudo-syntax: 

'> Avatar' {
  flex-shrink: 0
}